### ISSUE TITLE
Unknown option for rule MD052: shortcut-syntax

### ISSUE THREAD

**Issue by deiga** [rvben/rumdl#451]
### rumdl version

0.1.25

### Installation method

mise

### Bug description

[config warning] Unknown option for rule MD052: shortcut-syntax

### Steps to reproduce

1. Create an empty markdown file (or use any existing one): `touch test.md`
2. Configure `.rumdl.toml` with the below setting
3. Run `rumdl check test.md`

### Expected behavior

`rumdl` doesn't complain about documented configuration options

### Actual behavior

```
[config warning] Unknown option for rule MD052: shortcut-syntax

Success: No issues found in 1 file (13ms)
```

### Configuration

```toml
[MD052]
shortcut-syntax = true
```

### Operating system

macOS

**Comment by rvben** [comment]
Fixed in v0.1.26 @deiga. MD052 was missing a config section registration, so the validator didn't know about `shortcut-syntax`.
